### DISCUSSION TITLE
New 1.6 UI skin

### DISCUSSION THREAD

**Discussion by Haxardous** [multitheftauto/mtasa-blue#1054] [Ideas] [answered]
having a brand new official 1.6 skin would be excellent, as I see there's going to be a brand new community resources page coming up, having it fitting with it going to be perfect! (since the current one is probably uh.. aged well 👀 )

**Comment by prnxdev** [comment]
I would start by changing the GUI engine first.

**Comment by Lpsd** [comment]
I doubt the GUI library used by MTA will be changed. There have however been plans to upgrade it (newer versions of CEGUI offer much more features), which @patrikjuvonen was working on - as far as I know it's just the premake stuff which he needs some help with.

**Reply by Pirulax** [reply]
Yes Lopsi xD

**Comment by patrikjuvonen** [comment, marked as answer]
Resolved via https://github.com/multitheftauto/mtasa-blue/pull/2830 🎉 Thanks Hax
